cookbooks/aws-parallelcluster-common/spec/unit/resources/nfs_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+class ConvergeNfs
+  def self.setup(chef_run)
+    chef_run.converge_dsl('nfs::server', 'nfs::server4', 'nfs::_common') do
+      nfs 'setup' do
+        action :setup
+      end
+    end
+  end
+
+  def self.configure(chef_run)
+    chef_run.converge_dsl do
+      nfs 'configure' do
+        action :configure
+      end
+    end
+  end
+end
+
+describe 'nfs:setup' do
+  for_all_oses do |platform, version|
+    context "on #{platform}#{version}" do
+      let(:chef_run) do
+        ChefSpec::Runner.new(
+          platform: platform, version: version,
+          step_into: ['nfs']
+        )
+      end
+
+      let(:node) { chef_run.node }
+      let(:nfs_service) { platform == 'ubuntu' ? 'nfs-kernel-server.service' : 'nfs-server.service' }
+
+      before :each do
+        block_stepping_into_recipe
+      end
+
+      if %w(amazon centos redhat).include?(platform)
+        # It doesn't work because nfs::server fails to use `template` resource
+        # It should skip the entire recipe but it doesn't work.
+        #
+        # TODO: find a way to fix
+        #
+        # it 'installs nfs::server4' do
+        #   expect_to_include_recipe_from_resource('nfs::server')
```

Comment:
question, non-blocking, is nfs:server also for `amazon,centos,redhat` ? seems only for debian from the code

Reply:
Only debian systems need it.

Reply:
then why it's in the `amazon,centos,redhat` block?